apps/src/netsim/NetSimShardSelectionPanel.html.ejs

```diff
@@ -0,0 +1,37 @@
+<%
+  var i18n = require('../../locale/current/netsim');
+%>
+<div class="content-wrap">
+  <div class="field-box display-name-control">
+    <label for="netsim_lobby_name"><%= i18n.myName() %></label>
+    <input id="netsim_lobby_name" type="text" value="<%= displayName %>" <% if (displayName.length > 0) { %>disabled<% } %> />
```

Comment:
Aha: [This rule right here](http://google-styleguide.googlecode.com/svn/trunk/htmlcssguide.xml?showone=ID_and_Class_Name_Delimiters#ID_and_Class_Name_Delimiters)

I'll add a mention in our Code.org style guide under HTML/HAML